sos/plugins/postgresql.py

```diff
@@ -80,12 +80,19 @@ def setup(self):
 
 class RedHatPostgreSQL(PostgreSQL, SCLPlugin):
 
-    packages = ('postgresql', 'rh-postgresql95-postgresql-server', )
+    packages = (
+        'postgresql',
+        'rh-postgresql95-postgresql-server',
+        'rh-postgresql10-postgresql-server'
+    )
 
     def setup(self):
         super(RedHatPostgreSQL, self).setup()
 
-        scl = "rh-postgresql95"
+        if self.is_installed('rh-postgresql10-postgresql-server'):
+            scl = 'rh-postgresql10'
+        else:
+            scl = 'rh-postgresql95'
```

Comment:
Not hugely keen on this - can't we extend the `SCLPlugin` mechanism so that when a named instance (e.g. `rh-postgresql10-postgresql-server`) is present, that this information is exposed to the plugin somehow? That way we don't need to open-code version variation in every plugin that uses this.

Reply:
The plugin already has that method - the test can be safely replaced by

    if 'rh-postgresql10' in self._get_scls():

(but for future with more postgres versions in SCLs, we would have to come with smarter solution, like "give me highest/newest installed SCL version of postgres")

Reply:
Actually since #1567 was merged, we can do this based on the *active* scl, which is the desired end result anyways. I'll update this shortly.